### PR TITLE
ImpulseJointSet::get_mut option to wake up connected bodies

### PR DESCRIPTION
**Pull request by Vrixyz**
follow up to https://github.com/dimforge/rapier/issues/692

### CODE REVIEW

**Review by sebcrozet — CHANGES_REQUESTED** [review]
Thank you!

File: src/dynamics/joint/impulse_joint/impulse_joint_set.rs [hunk 1]
```diff
@@ -40,9 +42,11 @@ pub(crate) type JointGraphEdge = crate::data::graph::Edge<ImpulseJoint>;
 /// A set of impulse_joints that can be handled by a physics `World`.
 pub struct ImpulseJointSet {
     rb_graph_ids: Coarena<RigidBodyGraphIndex>,
-    joint_ids: Arena<TemporaryInteractionIndex>, // Map joint handles to edge ids on the graph.
+    /// Map joint handles to edge ids on the graph.
+    joint_ids: Arena<TemporaryInteractionIndex>,
     joint_graph: InteractionGraph<RigidBodyHandle, ImpulseJoint>,
-    pub(crate) to_wake_up: Vec<RigidBodyHandle>, // A set of rigid-body handles to wake-up during the next timestep.
+    /// A set of rigid-body handles to wake-up during the next timestep.
+    pub(crate) to_wake_up: HashMap<RigidBodyHandle, ()>,
```
Comment (sebcrozet) [inline src/dynamics/joint/impulse_joint/impulse_joint_set.rs:49]: Could you please open an issue on parry for adding `HashSet` ?

**Review by sebcrozet — APPROVED** [review]
Looking good now, thanks!

File: src/dynamics/joint/impulse_joint/impulse_joint_set.rs (hunk 1 above)
Comment (Vrixyz) [reply src/dynamics/joint/impulse_joint/impulse_joint_set.rs:49]: sure! 
- https://github.com/dimforge/parry/issues/269

**Pull request merged**